### PR TITLE
Improve youtube player background.

### PR DESCRIPTION
**Pull request by ebencarek**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,20 +22,20 @@
 			(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
 			(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
 			m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-			})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+		})(window,document,'script','http://www.google-analytics.com/analytics.js','ga');
```
Comment (mikeletscher) [inline index.html]: Any reason for this change?

Comment (killian-34) [reply index.html]: Keeping this background for the final landing page design

Comment (mikeletscher) [reply index.html]: I meant more the addition of `http`.

Comment (ebencarek) [reply index.html]: There was an error thrown in the browser because without the 'http:' it thinks it's a local path, not a url.

Comment (mikeletscher) [reply index.html]: Strange--double slash should inherit the protocol from the webpage.

I would suggest making this `https://`. Sorry for the nitpick.

**Pull request merged**